specs/howto.md

```diff
@@ -316,7 +316,7 @@ Examples of GeoJSON geometries are available
   [UN/CEFACT](https://www.unece.org/fileadmin/DAM/cefact/recommendations/rec20/rec20_rev3_Annex3e.pdf). 
   E.g.:
 
-  ```json
+```json
```

Comment:
so readthedocs does not support indentation for code blocks to align them with bullets... a pity

on the other side, i think we should add a short guide on how to local generate the readthedocs so to check that they work fine